packages/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
size of loader was incremented, but should be the same as it was
![image](https://user-images.githubusercontent.com/756731/85118256-c6aec580-b228-11ea-9b55-7ace7202cdb0.png)
![image](https://user-images.githubusercontent.com/756731/85118276-cb737980-b228-11ea-9068-d6e293f9d890.png)

Reply:
Previous implementation was incorrect. @kazurus, I can add padding to new implementation, to reproduce behavior without breaking changes

Reply:
@lamartire breaking change without padding

Reply:
Ok, then I will not do something

Reply:
maybe increase version to `0.19.0`? because this is breaking change

Reply:
This beaking change is not so big

Reply:
Anyway, we don't have a correct semver

Reply:
@lamartire please make 0.19